src/templates/error.js

```diff
@@ -6,11 +6,11 @@ export default (id, message, errorCode, code) => {
                 `<style>` +
                 `[id="${id}"].jw-error{background:#000;overflow:hidden;position:relative}` +
                 `[id="${id}"] .jw-error-msg{top:50%;left:50%;position:absolute;transform:translate(-50%,-50%)}` +
-                `[id="${id}"] .jw-error-text{color:#FFF;font:14px/1.35 Arial,Helvetica,sans-serif}` +
+                `[id="${id}"] .jw-error-text{text-align:start;color:#FFF;font:14px/1.35 Arial,Helvetica,sans-serif}` +
                 `</style>` +
                 `<div class="jw-icon jw-reset"></div>` +
                 `<div class="jw-info-container jw-reset">` +
-                    `<div class="jw-error-text jw-reset">${(message || '')}<span class="jw-break jw-reset"></span>${detail}</div>` +
+                    `<div class="jw-error-text jw-reset-text" dir="auto">${(message || '')}<span class="jw-break jw-reset"></span>${detail}</div>` +
```

Comment:
`class ="jw-error-text jw-reset-text" dir="auto"` is correct?

Reply:
good catch, that was meant to be in #3179